zipline/algorithm.py

```diff
@@ -1145,11 +1165,14 @@ def cancel_order(self, order_param):
 
     @api_method
     @require_initialized(HistoryInInitialize())
-    def history(self, assets, bar_count, frequency, field, ffill=True):
-        try:
-            assets = list(assets)
-        except TypeError:
-            assets = [assets]
+    def history(self, bar_count, frequency, field, ffill=True):
+        warnings.warn(
+            "The `history` method is deprecated.  Use `data.history` instead.",
+            category=ZiplineDeprecationWarning,
+            stacklevel=4
```

Comment:
well, the test currently verifies that we are getting to the proper stackframe to generate the warning, so I'm inclined to leave it alone for now.